site/en/blog/user-agent-reduction-origin-trial/index.md

```diff
@@ -131,25 +134,30 @@ provided that the permissions policy allows it.
         images or stylesheets) and subnavigations (for example, iframes) will send the
         reduced UA string, even if the origins of those requests are not
         enrolled in the origin trial.
-
+        
 1.  Load your website in Chrome M95 (or later) and start receiving the
     reduced UA string. 
 1.  Submit any issues or feedback to the UA Reduction [Github
     repository](https://github.com/abeyad/user-agent-reduction/issues).
 1.  See [https://uar-ot.glitch.me/](https://uar-ot.glitch.me/) for a simple
     demonstration of the origin trial (along with the source code).
 
-## How to participate in the origin trial as a third-party embed?
+## How to participate in the origin trial as a third-party embed? {: #enroll-third-party }
 
 Starting in Chrome 96, third-party embeds (for example, an iframe inside another site)
 can participate in the origin trial without requiring the top-level site to be
 enrolled.
 
 To enroll as a third-party embed, follow the same steps as for a [top-level site](#enroll-top-level),
```

Comment:
Wouldn't it be better to remove this sentence? Following the steps from above (i.e. adding the OT token via header) does not make sense for third-party enrollment, no?
As described below, the token needs to be injected via JavaScript.

Reply:
Thanks Sven, I updated the text, please let me know if it reads more clear.

Reply:
Yes, I do think this is better now. Thanks for fixing/rewriting this paragraph :-)